### PR TITLE
Working remote

### PR DESCRIPTION
**Pull request by ohenrik**
Just ment as a starting point, we should update this frequently as we learn more about the topic.

### CODE REVIEW

File: WORKING_REMOTE.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+We are all used to speaking with people who are not in the same
+physical place as us. Although we communicate a bitt less effectively over
+phone than when do in person it's usually not a big problem.
+
+However this communication challenge becomes much more apparent to us when
+we're trying to solve complex problems together.
+
+There are many things that makes communicating with remote people more
+difficult. Let's call them communication barriers.
```
Comment (ohenrik) [inline WORKING_REMOTE.md]: Change difficult to challenging. It is not necessarily more difficult, however the challenges are different from those we a re used to. Also our usual communication tool belt is not as well equipped to working remotely as working in the same physical location,

File: WORKING_REMOTE.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+bit.ly/ohenrik)
+
+\*Scheduling the meeting in a hyper account will make the hangout by default
+only allow access to people logged into an hyper.no email account.
+You can change this inside the hangout, but you have to do it every time you
+connect to the hangout, unless you use your private gmail account.
+
+## Why we all need to think remote
+
+*There is newer just one person working remotely in a team.*
```
Comment (jgorset) [inline WORKING_REMOTE.md]: Does do you mean by this? I mean, there could just be one person working remotely in a team… right? :grin:

File: WORKING_REMOTE.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+*   Good quality microphones (and enough of them)
+*   Good quality cameras (and enough of them)
+
+#### Permanent-Hangout
+
+To make a permanent hangout Follow these steps:
+
+1.  Schedule a meeting with your gmail account
+(preferably your private gmail account*). Be sure to create a hangout session
+that belongs to this meeting (should be on by default).
```
Comment (jgorset) [inline WORKING_REMOTE.md]: I think the meeting has to be in the future in order for the Hangout to not expire, so best make a meeting that happens in 40 years or something.

Comment (jgorset) [inline WORKING_REMOTE.md]: Fixed in 31e13a4f8c02e0b4cb3bed5277216b4efa171b3a.

**Comment by jgorset** [comment]
I added a section on sharing where we are!

File: WORKING_REMOTE.md (hunk 1 above)
Comment (jgorset) [inline WORKING_REMOTE.md]: :+1:

**Comment by timkurvers** [comment]
Awesome stuff! How do we deal with spelling fixes, comments inline?

**Comment by jgorset** [comment]
I like to just commit to the branch for uncontroversial changes, @timkurvers, but for discussing things I think inline comments are great.

**Comment by jgorset** [comment]
I started on your "why we all need to think remote"-section, @ohenrik, and it sorta grew into something that made sense as an introduction. What do you think?

**Comment by ohenrik** [comment]
The why we all need to think remote section was originally part of the introduction however i moved it further down because it was not finished :) Good addition!

**Comment by jgorset** [comment]
We didn't merge this already!?

**Comment by jgorset** [comment]
We get :cake:, @ohenrik!

**Comment by ohenrik** [comment]
:tada: :cake:

**Pull request merged**